posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -497,7 +497,7 @@ public class PostHog private constructor(
             if (values.contains(value)) {
                 shouldSendFeatureFlagEvent = false
             } else {
-                values.add(key)
+                values.add(value)
```

Comment:
`do not send feature flags called event twice` test already exists and it was failing correctly